index.md

```diff
@@ -19,11 +19,11 @@ status: editors-draft
   <h2 id="terms">Distinctions and Overlaps</h2>
   <dl>
     <dt id="accessibility"><strong>Accessibility</strong></dt>
-      <dd><em>Accessibility</em> addresses discriminatory aspects related to equivalent user experience for people with disabilities, including people with age-related impairments. For the web, accessibility means that people with disabilities can perceive, understand, navigate, and interact with websites and tools, and that they can contribute equally without barriers. For more information, see the <a href="https://www.w3.org/standards/webdesign/accessibility">Accessibility introduction</a>.</dd>
+      <dd><em>Accessibility</em> addresses discriminatory aspects related to equivalent user experience for people with disabilities. For the web, accessibility means that people with disabilities can perceive, understand, navigate, and interact with websites and tools, and that they can contribute equally without barriers. For more information, see the <a href="https://www.w3.org/standards/webdesign/accessibility">Accessibility introduction</a>.</dd>
```

Comment:
@susanatx where do you read "age as a subset of disability"? In the past, leaving out "age-related impairments" as a type of disability, led to people thinking that accessibility work is less relevant to designing for older people. I think this is reinforcing your point, that "Disabilities can, and are likely, to come with age but being aged in and of itself is not a disability".